test/testsyncmove.cpp

```diff
@@ -33,6 +32,11 @@ struct OperationCounter {
             return nullptr;
         };
     }
+
+    friend inline QDebug operator<<(QDebug dbg, const OperationCounter &oc)
```

Comment:
off topic but I was recently wondering which version the preferable is.

`QDebug &operator<<(QDebug &dbg,`
 or 
`QDebug operator<<(QDebug &dbg,`